app/src/main/java/com/isc/hermes/MainActivity.java

```diff
@@ -103,19 +105,27 @@ protected void onCreate(Bundle savedInstanceState) {
 
     public void testPolyline() { // this is a test method that will be removed once the functionality has been verified.
         Map<String, String> r = new HashMap<>();
+        List<String> routes = new ArrayList<>();
+        List<Integer> colors = new ArrayList<>();
 
         r.put("Route A", "{\"type\":\"Feature\",\"distance\":0.5835077072636502,\"properties\":{},\"geometry\":{\"type\":\"LineString\",\"coordinates\":[[-66.156338,-17.394251],[-66.155208,-17.394064],[-66.154149,-17.393858],[-66.15306,-17.393682],[-66.15291,-17.394716],[-66.153965,-17.394903]]}}");
         r.put("Route B", "{\"type\":\"Feature\",\"distance\":0.5961126697414532,\"properties\":{},\"geometry\":{\"type\":\"LineString\",\"coordinates\":[[-66.156338,-17.394251],[-66.155208,-17.394064],[-66.155045,-17.39503],[-66.154875,-17.396151],[-66.153754,-17.395951],[-66.153965,-17.394903]]}}");
-        r.put("Route C", "{}");
+        r.put("Route C", "{\"type\":\"Feature\",\"distance\":0.5961126697414532,\"properties\":{},\"geometry\":{\"type\":\"LineString\",\"coordinates\":[[-66.156338,-17.394251],[-66.155208,-17.394064],[-66.155045,-17.39503],[-66.154875,-17.396151],[-66.153754,-17.395951],[-66.153965,-17.394903]]}}");
 
         String jsonA = r.get("Route A");
         String jsonB = r.get("Route B");
         String jsonC = r.get("Route C");
 
-        MapPolyline mapPolyline = new MapPolyline(mapView);
-        //mapPolyline.displaySavedCoordinates(jsonB, Color.RED);
-        mapPolyline.displaySavedCoordinates(jsonA, Color.BLUE);
+        routes.add(jsonA);
+        routes.add(jsonB);
+        routes.add(jsonC);
+
+        colors.add(0xFF0000FF);
```

Comment:
I have doubts about how the colors are assigned since I see that colors are assigned in the test and should be assigned automatically.

Reply:
is a test method so for the moment it will ber harcoded